nixos/modules/system/boot/systemd-nspawn.nix

```diff
@@ -42,8 +42,61 @@ let
     (assertValueOneOf "VirtualEthernet" boolValues)
   ];
 
+  mounts = mkOption {
+    type = types.attrsOf (types.submodule {
+      options = {
+        source = mkOption {
+          type = types.str;
+          default = "";
+          description = "mount source path";
+        };
+        mountOption = mkOption {
+          type = types.str;
```

Comment:
Would it make sense to use `types.enum []` here?

Reply:
I came back. we may specify like 'mount=700,uid=0,gid=0' as a mount options of tmpfs, so enum types would not fit here.